keanu-project/src/main/java/io/improbable/keanu/tensor/dbl/DoubleTensor.java

```diff
@@ -92,6 +93,21 @@ static DoubleTensor scalar(double scalarValue) {
         return new ScalarDoubleTensor(scalarValue);
     }
 
+    /**
+     * @param toPile    an array of DoubleTensor
+     * @return  a DoubleTensor with toPile joined along a new dimension 0
```

Comment:
An example here would be helpful

Reply:
and/or an explanation of how this differs from `concat` (one creates an extra dimension, the other doesn't, right?)

Reply:
Yup. Sounds good.